lit-actions/src/index.ts

```diff
@@ -27,8 +27,10 @@ const main = async () => {
     const duration = endTime - startTime;
     console.log("🕒 Duration:", duration / 1000, "seconds");
     console.log("🔗 IPFS CID:", result.IpfsHash);
-    results[file] = {
+    const name = file.replace(/.js$/, ""); // invoke actions as name only, without name.js
```

Comment:
callout

Reply:
okay, noted
this might need some attention in `telegram.service.ts` in the `/lit` section. will take a look at it.